### PR TITLE
IA-1022 Lqas/IM send country to api

### PR DESCRIPTION
**Pull request by quang-le**
## Changes

- modify `getLqasIm` to take a countryId param and pass it through the query
- adapt query hooks accordingly
- pass `country` param to all components using query hooks
- `api.py`: only loop through forms from country that was passed through request
- Campaigns select box: adapt loading prop to be active on `isFetching`

### CODE REVIEW

File: plugins/polio/js/src/pages/IM/index.js [hunk 1]
```diff
@@ -42,16 +42,17 @@ export const ImStats = ({ imType }) => {
     const classes = useStyles();
     const [campaign, setCampaign] = useState();
     const [country, setCountry] = useState();
-    const { data: imData, isLoading } = useLqasIm(imType);
+    const { data: imData, isLoading, isFetching } = useLqasIm(imType, country);
```
Comment (beygorghor) [inline plugins/polio/js/src/pages/IM/index.js:45]: Why use isLoading and isFetching ? I would use only isFetching.

File: plugins/polio/js/src/pages/IM/requests.js [hunk 2]
```diff
@@ -3,49 +3,55 @@ import { getRequest } from 'Iaso/libs/Api';
 import { IM_POC_URL, LQAS_POC_URL } from './constants.ts';
 import { convertAPIData } from '../../utils/LqasIm.tsx';
 
-export const getLqasIm = type => {
-    if (type === 'imOHH') return getRequest(`${IM_POC_URL}?type=OHH`);
-    if (type === 'imIHH') return getRequest(`${IM_POC_URL}?type=HH`);
-    if (type === 'imGlobal') return getRequest(`${IM_POC_URL}`);
-    if (type === 'lqas') return getRequest(`${LQAS_POC_URL}`);
+export const getLqasIm = (type, countryId) => {
+    if (type === 'imOHH')
```
Comment (beygorghor) [inline plugins/polio/js/src/pages/IM/requests.js:7]: this is the use case of a switch here no ?

**Review by beygorghor — APPROVED** [review]
Nothing blocking.
Just noticed you spread the country id everywhere. 
In this case redux can be a good solution.

File: plugins/polio/js/src/pages/IM/requests.js (hunk 2 above)
Comment (quang-le) [reply plugins/polio/js/src/pages/IM/requests.js]: true, although I like the simplicity of `if... return`

**Comment by quang-le** [comment]
> Nothing blocking. Just noticed you spread the country id everywhere. In this case redux can be a good solution.

Don't say that word 😉. `Context` api would work just as well, and  with way less boilerplate 😁

**Pull request merged**